gate-web/src/test/resources/gate-test.yml

```diff
@@ -20,3 +20,21 @@ services:
   mine.enabled: false
 
   swabbie.enabled: false
+
+---
+
+spring:
+  profiles: alloworigincors
+
+cors:
+  allow-mode: "originlist"
+  allowed-origins: "https://api.spinnaker.netflix.net, https://atlas.netflix.net"
```

Comment:
does this also work as yaml configuration with a list of values?
```
  allowed-origins:
    - https://api.spinnaker.netflix.net
    - https://atlas.netflix.net
```
?

Reply:
That format requires a Configuration object representing that structure. Didn't see value adding a ConfigProp value object.  This works if you are thinking of readability while adding several domains:
```
cors:
  allow-mode: "originlist"
  allowed-origins: >
    https://api.spinnaker.netflix.net,
    https://atlas.netflix.net 
```

Reply:
ah ok I wasn't sure if an `@Value` would deserialize the yaml list onto a List parameter, and the syntax you showed looks fine to me